packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -335,6 +335,27 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
               teamId: eventTypeWorkflow.teamId,
               verifiedAt: step.verifiedAt,
             });
+          } else if (step.action === WorkflowActions.SMS_ATTENDEE) {
+            const attendee = booking.attendees[0]; // Use the first attendee if multiple
+            if (attendee?.phoneNumber) {
+              await scheduleSMSReminder({
+                evt: bookingInfo,
+                reminderPhone: attendee.phoneNumber,
```

Comment:
This is not the correct phone number. You can find the right phone number on the booking smsReminderNumber